tests/System.Text.Primitives.Tests/PrimitiveParserUInt32PerfTests.cs

```diff
@@ -279,23 +279,28 @@ private unsafe static void PrimitiveParserByteSpanToUInt32(string text)
             }
         }
 
-        [Benchmark(Skip = "The generic type 'System.Collections.Generic.List`1' was used with an invalid instantiation in assembly 'System.Private.CoreLib")]
-        private unsafe static void PrimitiveParserByteSpanToUInt32_VariableLength()
+        [Benchmark]
+        [InlineData(0)]
```

Comment:
I tried that. You cannot create an array of Span.
ReadOnlySpan<byte>[] byteSpanList = new ReadOnlySpan<byte>[10];

..\..\dotnet\dotnet.exe run -c Release

Unhandled Exception: System.TypeLoadException: Could not create array type 'System.ReadOnlySpan`1[System.Byte][]' from assembly 'System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e'.
   at ConsoleApp.Program.Main(String[] args)